connectors/src/main/kotlin/fund/cyber/markets/connectors/bitstamp/BitstampMessageParser.kt

```diff
@@ -0,0 +1,56 @@
+package fund.cyber.markets.connectors.bitstamp
+
+import com.fasterxml.jackson.databind.JsonNode
+import fund.cyber.markets.connectors.common.ContainingUnknownTokensPairMessage
+import fund.cyber.markets.connectors.common.ExchangeMessage
+import fund.cyber.markets.connectors.common.TradesAndOrdersUpdatesMessage
+import fund.cyber.markets.connectors.common.ws.SaveExchangeMessageParser
+import fund.cyber.markets.connectors.common.ws.pusher.getPusherEventByValue
+import fund.cyber.markets.connectors.jsonParser
+import fund.cyber.markets.model.TokensPair
+import fund.cyber.markets.model.Trade
+import fund.cyber.markets.model.TradeType
+import java.math.BigDecimal
+import java.util.*
+
+private val event_type_trade = "trade"
+
+open class BitstampMessageParser(
+        private val channelIdForTokensPairs: Map<String, TokensPair>
+): SaveExchangeMessageParser() {
+
+    override fun parseMessage(jsonRoot: JsonNode): ExchangeMessage? {
+        val eventType = jsonRoot["event"].asText()
+
+        val pusherEvent = getPusherEventByValue(eventType)
```

Comment:
Can be wrote as:
```
val pusherEvent = getPusherEventByValue(eventType) ?: return pusherEvent.parseMessage(jsonRoot)
```

instead of following if check